load() in control/control_test.go causes tests to panic if plugin doesn't exist

If the mock2 plugin doesn't exist the mock load function panics. 

```
--- FAIL: TestSwapPlugin (0.10s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x993a8]

goroutine 12 [running]:
testing.tRunner.func1(0xc8200a4b40)
	/Users/tjerniga/.gvm/gos/go1.5/src/testing/testing.go:450 +0x171
github.com/intelsdi-x/snap/control.(*pluginControl).returnPluginDetails(0xc8200aa9a0, 0x0, 0x0, 0x0, 0x0)
	github.com/intelsdi-x/snap/control/_test/_obj_test/control.go:339 +0xb78
github.com/intelsdi-x/snap/control.(*pluginControl).Load(0xc8200aa9a0, 0x0, 0x0, 0x0, 0x0, 0x0)
	github.com/intelsdi-x/snap/control/_test/_obj_test/control.go:246 +0x151
github.com/intelsdi-x/snap/control.load(0xc8200aa9a0, 0xc820055e28, 0x1, 0x1, 0x0, 0x0, 0x0, 0x0)
	/Users/tjerniga/.gvm/pkgsets/system/global/src/github.com/intelsdi-x/snap/control/control_test.go:89 +0x167
github.com/intelsdi-x/snap/control.TestSwapPlugin(0xc8200a4b40)
	/Users/tjerniga/.gvm/pkgsets/system/global/src/github.com/intelsdi-x/snap/control/control_test.go:126 +0x225
testing.tRunner(0xc8200a4b40, 0x948500)
```

--- a/scripts/test.sh
+++ b/scripts/test.sh
@@ -25,6 +25,12 @@
 # 5. race detector (http://blog.golang.org/race-detector)
 # 6. test coverage (http://blog.golang.org/cover)
 
+# If the following plugins don't exist, exit
+[ -f $SNAP_PATH/plugin/snap-collector-mock1 ] || { echo 'Error: $SNAP_PATH/plugin/snap-collector-mock1 does not exist. Run make to build it.' ; exit 1; } 
+[ -f $SNAP_PATH/plugin/snap-collector-mock2 ] || { echo 'Error: $SNAP_PATH/plugin/snap-collector-mock2 does not exist. Run make to build it.' ; exit 1; } 
+[ -f $SNAP_PATH/plugin/snap-processor-passthru ] || { echo 'Error: $SNAP_PATH/plugin/snap-processor-passthru does not exist. Run make to build it.' ; exit 1; } 
+[ -f $SNAP_PATH/plugin/snap-publisher-file ] || { echo 'Error: $SNAP_PATH/plugin/snap-publisher-file does not exist. Run make to build it.' ; exit 1; } 
+
 TEST_DIRS="cmd/ control/ core/ mgmt/ pkg/ snapd.go scheduler/"
 VET_DIRS="./cmd/... ./control/... ./core/... ./mgmt/... ./pkg/... ./scheduler/... ."
 

